cpp/test/Slice/errorDetection/ChangedMeaning.err

```diff
@@ -8,7 +8,6 @@ ChangedMeaning.ice:79: 'c2' has changed meaning
 ChangedMeaning.ice:96: 'CounterSeq' has changed meaning
 ChangedMeaning.ice:105: redefinition of parameter 'param'
 ChangedMeaning.ice:123: 'E' is not an exception
-ChangedMeaning.ice:123: 'E' has changed meaning
```

Comment:
The real error is that `E` was used as an exception when it is not, not that it changed meaning.